### ISSUE TITLE
Students cannot starting SEB V3.1.1

### ISSUE THREAD

**Issue by vanwelie** [SafeExamBrowser/seb-win-refactoring#73]
Hi there!

About a hundred students, spread over 5 classes, experienced problems opening the SEB after updating to version 3.1.1 on a Windows 10 device.

This problem occurs at various agencies that offer online tests (Woots.nl/Learnbeat.nl) in conjunction with the SEB.
In the annex some log files from one device, on which the SEB cannot be started.

I've been researching the SEB itself. I think there's a mistake there. And especially in the default browser settings. Because with a completely clean SEB configuration from the SEB Configuration Tool, SEB doesn't open the file either.

I once checked "Use SEB without browser window" in the "Browser" tab at the bottom, and then SEB opens the configuration, but I only get to see the SEB taskbar and a black screen. So I think the "User agent for desktop mode > desktop default" is not configured correctly by SEB. Can you get into that, too?

If so, how can I fix it? Because I don't know to fix it =D
If not, I don't know what's causing it.

Greetings
Dennis van Welie

Attached some of the error messages we get:
[2021-02-09_23h55m18s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/5971876/2021-02-09_23h55m18s_Runtime.log)
[2021-02-09_23h55m18s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/5971877/2021-02-09_23h55m18s_Client.log)
![20210209_224319](https://user-images.githubusercontent.com/49212758/107771118-975e4300-6d3a-11eb-9744-405bd6d433e8.jpg)
[SEB-settings give error.zip](https://github.com/SafeExamBrowser/seb-win-refactoring/files/5971898/SEB-settings.give.error.zip)

**Comment by SebastianZiegler** [comment]
We had the same issue.

You can solve this by installing the following pakages first and then reinstall SEB:

.NET Framework 4.7.2 Runtime: https://dotnet.microsoft.com/download/dotnet-framework/net472
Visual C++ 2015-2019 Redistributable: https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads

**Comment by vanwelie** [comment]
Hi SebastianZiegler,

Thanks for your quick respons.
I have try the solution.

.NET Framework 4.7.2 Runtime I cannot install, because I have a newer version already installed on my desktop.
I have installed Visual C++ and restart my device. When I try again, I get the same notification.

This not solved the problem =(
Do you have other ideas?

Thanks a lot!
Dennis van Welie

**Comment by dbuechel** [comment]
You need to make sure that you install the correct bitness of the VC++ redistributable: If you installed SEB for 32-bit systems (x86), you'll need to install the 32-bit redistributable (x86). Likewise, if you installed the 64-bit build of SEB (x64), you'd need to install the 64-bit redistributable (x64).

**Comment by dbuechel** [comment]
x86: https://aka.ms/vs/16/release/vc_redist.x86.exe
x64: https://aka.ms/vs/16/release/vc_redist.x64.exe

**Comment by vanwelie** [comment]
I have install the right redistributable VC++, but I got still errors: see below.

![210213 SEB error 01](https://user-images.githubusercontent.com/49212758/107845590-d7760200-6ddc-11eb-90a0-6895394b7f78.jpg)
![210213 SEB error 02](https://user-images.githubusercontent.com/49212758/107845594-db098900-6ddc-11eb-8bce-647d1d26911b.jpg)

**Comment by vanwelie** [comment]
And here the log file ... 
[2021-02-13_09h18m42s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/5975335/2021-02-13_09h18m42s_Runtime.log)

**Comment by vanwelie** [comment]
I have the problem solved.
After installing the programs from @SebastianZiegler , I removed SEB with Revo Uninstaller and restart my device. Then, I downloaded SEB again from safeexambrowser.org and install it as an admin user.

Now it works!

I think the problem is to update from version 2.4.1 to version 3.1.1, without uninstall the last version. 
I start to write an manual for my students and teachers. I hope that fixed the problem for everyone.

Thanks for thinking with me!

Regards
Dennis van Welie
